invenio_records_lom/fixtures/demo.py

```diff
@@ -0,0 +1,363 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 Graz University of Technology.
+#
+# invenio-records-lom is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+import json
+
+from faker import Faker
+
+
+# ----- functions for LOM datatypes -----
+def langstringify(fake: Faker, string: str) -> dict:
+    """Turn `string` into a LangString-object, as specified by LOMv1.0-standard."""
+    return {
+        "language": create_fake_language(fake),
+        "string": string,
+    }
+
+
+def vocabularify(fake: Faker, choices: list) -> dict:
+    """Randomly draw a choice from `choices`, then turn that choice into a Vocabulary-object, as specified by LOMv1.0-standard"""
+    return {
+        "source": "LOMv1.0",
+        "value": fake.random.choice(choices),
+    }
+
+
+def create_fake_datetime(fake: Faker) -> dict:
+    """Create a fake DateTime-object, compatible with LOMv1.0-standard."""
+    pattern = fake.random.choice(["YMDhmsTZD", "YMDhms", "YMD", "Y"])
+    if pattern == "Y":
+        datetime = fake.year()
+    elif pattern == "YMD":
+        datetime = fake.date()
+    elif pattern == "YMDhms":
+        datetime = fake.date_time().isoformat()
+    elif pattern == "YMDhmsTZD":
+        tzd = fake.random.choice(["UTC", "CST", "JST"])
+        datetime = fake.date_time().isoformat() + tzd
+
+    return {"dateTime": datetime, "description": langstringify(fake, fake.sentence())}
+
+
+def create_fake_duration(fake: Faker) -> dict:
+    """Create a fake Duration-object, compatible with LOMv1.0-standard."""
+    random = fake.random
+    pattern = random.choice(["all", "Y", "D", "HM", "S"])
+    if pattern == "all":
+        duration = "P1Y2M4DT10H35M12.5S"
+    elif pattern == "Y":
+        duration = f"P{random.randint(1,5)}Y"
+    elif pattern == "D":
+        duration = f"P{random.randint(1,60)}D"
+    elif pattern == "HM":
+        duration = f"PT{random.randint(1,3)}H{random.randint(1,59)}M"
+    else:
+        duration = f"PT{random.uniform(0.1, 12.5)}S"
+
```

Comment:
Suggestion: to write it like the following

random = fake.random
pattern = random.choice(["all", "Y", "D", "HM", "S"])
duration = {
  "all": "P1Y2M4DT10H35M12.5S",
  "Y": f"P{random.randint(1,5)}Y",
  "D": f"P{random.randint(1,60)}D",
  "HM": f"PT{random.randint(1,3)}H{random.randint(1,59)}M",
  "S": f"PT{random.uniform(0.1, 12.5)}S",
}

return {
  "duration": duration[pattern],
  "description": langstringify(fake, fake.sentence()),
}